trainer/src/trainer/main.py

```diff
@@ -80,39 +83,6 @@ async def training_status(
         )
 
 
-@stub.function(
```

Comment:
How are we now persisting non-fireworks models?

Reply:
We're still uploading the LoRAs to s3. These changes will make it so we can't fall back on pipeline2 to serve the LoRAs anymore, but we can still fall back to pipeline3 on Modal.